src/App.js

```diff
@@ -1,26 +1,70 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import foods from './foods.json'
+import FoodBox from './components/FoodBox';
+import AddButton from './components/AddButton'
+import 'bulma/css/bulma.css'
+import AddForm from './components/AddForm'
+import Search from './components/Search'
+import TodayList from './components/TodayList'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state= {
+      foods: foods,
+      filteredFoods: foods,
+      todayFoods:[],
+      displayForm: false
+    }
+  }
+
+  toggleForm=()=>{
+    let formState = this.state.displayForm
+    formState = !formState
+    this.setState({displayForm: formState})
```

Comment:
you could reduce code here, just doing this single line

`this.setState({displayForm: !this.state.displayForm})`